src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/chapterPage/viewModel/ChapterPageViewModel.kt

```diff
@@ -0,0 +1,58 @@
+package org.wycliffeassociates.otter.jvm.app.ui.chapterPage.viewModel
+
+import io.reactivex.rxkotlin.subscribeBy
+import javafx.collections.FXCollections
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.ChapterPageModel
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.Verse
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import tornadofx.*
+
+class ChapterPageViewModel : ViewModel() {
+    val model = ChapterPageModel()
+//    val chapterPageUseCase = ChapterPageUseCase(Injector.chapterDao)
+    private val modelChapters = model.chapters
+    val activeChapter = model.activeChapter
+    val selectedTab = model.selectedTab
```

Comment:
Would it be easier to just use `model.chapters`, etc., in the view model instead of declaring new variables here?

Reply:
it might be. I just thought that it would look cleaner at the end of the day. but they end up being the same length anyway, so i'm not really opposed to changing it back